src/cf-forms.less

```diff
@@ -56,15 +60,31 @@
         <label class="form-label-header">
             Form label header
         </label>
+    - name: Form label helper text
+      markup: |
+        <label class="form-label-helper-text">
+            Form label helper text
+        </label>
   tags:
     - cf-forms
 */
 
 .form-label-header {
-    .h5();
+    .h4();
     margin-bottom: unit(10px / @font-size, em);
 }
 
+.form-label-helper-text {
+    .micro-copy();
+    color: @label-helper;
+    // 14px font-size value is from .micro-copy
```

Comment:
@Scotchester should we update `.micro-copy()` to expose a `@font-size` variable like the headings to avoid needing to leave notes like this?